<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="ipAddrLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ipAddrLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server IP Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="serverConnectPushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect!</string>
    </property>
   </widget>
   <widget class="QLabel" name="logLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="logTextEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>331</width>
      <height>361</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="serverDisconnectPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serialPortLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="serialPortLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Port</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="serialConnectPushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serialDisconnectPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
